students/20th/wonyoungcho/westagram/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang = 'en'>
+<head>
+    <meta charset = 'UTF-8'>
+    <meta name = 'viewport' content = 'width=device-width, initial-scale=1.0'>
+    <title>westagram</title>
+    <link rel = 'stylesheet' href = 'css/style.css' />
+</head>
+<body>
+     <main class = 'mainLogin'>
+        <div class = 'loginBox'>
```

Comment:
```suggestion
     <main class='mainLogin'>
        <div class='loginBox'>
```
저는 등호와 프로퍼티를 붙여 쓰는 쪽이 좋아 보이는데 어떻게 생각하세요?